samples/nrf9160/modem_shell/src/gnss/gnss_shell.c

```diff
@@ -184,44 +209,92 @@ static int cmd_gnss_mode_periodic_gnss(const struct shell *shell, size_t argc, c
 	return err;
 }
 
-static int cmd_gnss_config(const struct shell *shell, size_t argc, char **argv)
+static int cmd_gnss_dynamics(const struct shell *shell, size_t argc, char **argv)
 {
 	return print_help(shell, argc, argv);
 }
 
-static int cmd_gnss_config_startmode(const struct shell *shell, size_t argc, char **argv)
+static int cmd_gnss_dynamics_general(const struct shell *shell, size_t argc, char **argv)
 {
-	return print_help(shell, argc, argv);
+	if (!gnss_running) {
+		shell_error(shell, "%s: start GNSS to execute command", argv[0]);
+		return -ENOEXEC;
+	}
+
+	int err;
+
+	err = gnss_set_dynamics_mode(GNSS_DYNAMICS_MODE_GENERAL);
```

Comment:
This could be simply `return gnss_set_dynamics_mode(GNSS_DYNAMICS_MODE_GENERAL);`, same in other functions below.

Reply:
I guess those were leftovers from earlier when gnss_shell.c would also print errors. There were a lot of those, but all should now be fixed.